share/docker/openfast_ubuntu/Dockerfile

```diff
@@ -13,42 +12,63 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+ARG BASE=ubuntu:mantic
```

Comment:
Should we stick with LTS releases (Jammy) instead of shorter support ones?

Reply:
Yeah that makes sense. I'll switch to `jammy` tomorrow and verify that it builds

Reply:
The image builds fine with `jammy` although I've just noticed that the version `openfast` reports when I build from the `v3.5.2` tag is ` OpenFAST-v3.5.2-dirty` - any idea why it says it's dirty?

Reply:
We haven't set the tag for 3.5.3 yet -- we do that after the merge to main.  I wonder if this will cause an issue for correctly setting the build info on automated builds after merges to main if the latest tag doesn't yet exist.

Not sure why it shows as dirty and doesn't include the number of commits since 3.5.2

Reply:
If the release candidate branches are always named `rc-major.minor.patch`, the tag for the docker image should be fine as I'm extracting the version from the branch name. However:
- The `openfast` CLI in the docker image won't show the version on the command line as (I think) it extracts it from the git ref
- The docker tag will be wrong if you decide to use a different git tag after merging

We could fix this by adding a `VERSION` file or something similar to `package.json` or `pyproject.toml`. You'd update it with the new version number for openfast before merging. The docker workflow would then use this for the docker tag and the openfast CLI would also need to to show the version.

Doing this would also open up the possibility for automating tags/releases with a workflow so the git tag is always created before the docker image is built and you don't have to manually release. You could either add the release notes as usual or use some nice automation to pull the release notes from somewhere else (like the pull request description or commits)

Reply:
Good thoughts!  We've typically been doing release merges directly from `dev`, but that would be simple to always do an `rc-x.y.z` branch first for the release.  We can probably tag directly from the last commit on the `rc` branch prior to merging to `main`.  This approach could in theory eliminate the issue of getting the correct tag into docker.

Reply:
Yeah that would work I think! As long as you're happy with the tag being on the commit before the merge commit on `main`

Reply:
I think it is acceptable.  We'll do a merge from `main` into the `rc-x.y.z` branch right before the final merge to `main`.  I suspect we'll end up with the same git commit hash on `main` that way.